account_reconcile_oca/models/account_move_line.py

```diff
@@ -1,14 +1,22 @@
 # Copyright 2023 Dixmit
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _, models
+from odoo import _, fields, models
 from odoo.exceptions import ValidationError
 
 
 class AccountMoveLine(models.Model):
 
     _inherit = "account.move.line"
 
+    invoice_due_date = fields.Date(
+        related="move_id.invoice_date_due",
+        string="Invoice Due Date",
+        readonly=True,
+        store=True,
```

Comment:
You don't need to make it stored

Reply:
@ACheung-FactorLibre can you make it not readable please?

Reply:
Done!